providers/fips/fipsprov.c

```diff
@@ -1238,3 +1259,128 @@ void OSSL_INDICATOR_get_callback(OSSL_LIB_CTX *libctx,
             *cb = NULL;
     }
 }
+
+/* On Deamnd test infrastructure */
+
+struct ondemand_test_status {
+    const char *alg;
+    int pass;
+};
+
+static struct ondemand_test_status ondemand_kat_integrity_tests[] = {
+    { NULL, 0 }
+};
+static struct ondemand_test_status ondemand_kat_cipher_tests[] = {
+    { NULL, 0 }
+};
+static struct ondemand_test_status ondemand_kat_asym_cipher_tests[] = {
+    { NULL, 0 }
+};
+static struct ondemand_test_status ondemand_kat_asym_keygen_tests[] = {
+    { "SLH-DSA-SHA2-128f", 0 },
+    { NULL, 0 }
+};
+static struct ondemand_test_status ondemand_kat_kem_tests[] = {
+    { NULL, 0 }
+};
+static struct ondemand_test_status ondemand_kat_digest_tests[] = {
+    { NULL, 0 }
+};
+static struct ondemand_test_status ondemand_kat_signature_tests[] = {
+    { "SLH-DSA-SHAKE-128f", 0 },
```

Comment:
With deferred testing, it seems reasonable to include `SLH-DSA-SHAKE-128s` as well.  This is recommended by the standards but not mandated.